.travis.yml

```diff
@@ -3,13 +3,15 @@ language: python
 python:
   - "3.4"
   - "3.5"
+  - "3.6"
+  - "3.7-dev"
```

Comment:
Please change to what I recommended in #144 (stable 3.7.0)

Reply:
I'll just abandon this PR for now and try again after we get 3.7 Travis running on master.

Reply:
Adding it with allowed failures allows everyone see the status and pick up some tests for fixing. Thus it would be beneficial :)